src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -219,6 +256,14 @@ private void importFromGcs(String projectId, TableReference tableRef, @Nullable
 
       // Poll until job is complete.
       waitForJobCompletion(bigQueryHelper.getRawBigquery(), projectId, jobReference);
+
+      if (temporaryTableReference != null && bigQueryHelper.tableExists(temporaryTableReference)) {
+        long expirationMillis = DateTime.now().plusDays(1).getMillis();
+        Table table = bigQueryHelper.getTable(temporaryTableReference).setExpirationTime(expirationMillis);
+        bigQueryHelper.getRawBigquery().tables().update(temporaryTableReference.getProjectId(),
```

Comment:
is there a way to specify this in the load config rather than updating it later? Otherwise a pipeline failure before this point will cause the temporary table to not have the expiration set.

Reply:
Load config does not have possibility to configure expiration time.